.travis.yml

```diff
@@ -36,6 +36,8 @@ env:
     - TRAVIS_FLAVOR=mongo FLAVOR_VERSION=3.0.1
     - TRAVIS_FLAVOR=mongo FLAVOR_VERSION=3.2.10
     - TRAVIS_FLAVOR=fluentd FLAVOR_VERSION=v0.12.22
+    - TRAVIS_FLAVOR=nginx FLAVOR_VERSION=1.6.2
+    - TRAVIS_FLAVOR=nginx FLAVOR_VERSION=1.7.11
```

Comment:
I'm only aiming for parity with the agent. I'm up to add to it as well, where it's called for.